client/public/index.html

```diff
@@ -7,6 +7,10 @@
     <title>Code Friender</title>
   </head>
   <body>
+    <style>
+      body {background-image: linear-gradient(#415591,#f8f4e4)}
```

Comment:
we should try and keep styling out of the public index.html file
You think you can add this to the root css?